server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -443,11 +443,12 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
 
     try {
       CheckCompactionConfig.validate(manager.getConfiguration());
-    } catch (SecurityException | IllegalArgumentException | IllegalStateException
-        | ReflectiveOperationException e) {
+      this.metrics.clearCompactionServiceConfigurationError();
+    } catch (RuntimeException | ReflectiveOperationException e) {
```

Comment:
I changed this to catch RuntimeException because JSON parsing exceptions were also being thrown from the tests.